src/method/method_struct_query.cpp

```diff
@@ -493,7 +493,7 @@ namespace cubmethod
     fprintf (stdout, "tuple_count: %d\n", tuple_count);
     fprintf (stdout, "ins_oid (%d, %d, %d)\n", ins_oid.pageid, ins_oid.slotid, ins_oid.volid);
     fprintf (stdout, "include_oid: %d\n", include_oid);
-    fprintf (stdout, "query_id: %lld\n", query_id);
+    fprintf (stdout, "query_id: %ud\n", (unsigned int)query_id);
```

Comment:
query_id is of type UINTPTR.
This is defined differently depending on define.
Therefore, if you do it like below, you will be able to see the warning again even if the type changes in the future.

fprintf (stdout, "query_id: %ud\n", query_id);

Reply:
It must be %u rather than %ud.
Note that the following code prints '123d'
```
#include <stdio.h>

int
main() {
    printf("%ud\n", 123);
}
```